src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -39,16 +39,16 @@ public boolean extractQuestionDetails(Map<String, String[]> requestParameters,
         super.extractQuestionDetails(requestParameters, questionType);
 
         String minRecipientsToBeRanked = HttpRequestHelper.getValueFromParamMap(
-                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANKMINRECIPIENTSTOBERANKED);
+                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANK_MIN_RECIPIENTS_TO_BE_RANKED);
         String maxRecipientsToBeRanked = HttpRequestHelper.getValueFromParamMap(
-                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANKMAXRECIPIENTSTOBERANKED);
-        String isMinRecipientsToBeRankedEnabled = HttpRequestHelper.getValueFromParamMap(
-                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANKISMINRECIPIENTSTOBERANKEDENABLED);
-        String isMaxRecipientsToBeRankedEnabled = HttpRequestHelper.getValueFromParamMap(
-                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANKISMAXRECIPIENTSTOBERANKEDENABLED);
+                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANK_MAX_RECIPIENTS_TO_BE_RANKED);
+        String minRecipientsToBeRankedEnabled = HttpRequestHelper.getValueFromParamMap(
+                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANK_IS_MIN_RECIPIENTS_TO_BE_RANKED_ENABLED);
+        String maxRecipientsToBeRankedEnabled = HttpRequestHelper.getValueFromParamMap(
+                requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_RANK_IS_MAX_OPTIONS_TO_BE_RANKED_ENABLED);
 
-        this.minOptionsToBeRankedEnabled = "on".equals(isMinRecipientsToBeRankedEnabled);
-        this.maxOptionsToBeRankedEnabled = "on".equals(isMaxRecipientsToBeRankedEnabled);
+        this.isMinOptionsToBeRankedEnabled = "on".equals(minRecipientsToBeRankedEnabled);
+        this.isMaxOptionsToBeRankedEnabled = "on".equals(maxRecipientsToBeRankedEnabled);
 
         if (minRecipientsToBeRanked != null && !"".equals(minRecipientsToBeRanked)) {
             this.minOptionsToBeRanked = Integer.parseInt(minRecipientsToBeRanked);
```

Comment:
`this.minOptionsToBeRanked` should not be shadowed by `minOptionsToBeRanked` which can be named better since it is a parameter.

Also, there's an error: this shouldn't be `minRecipientsToBeRanked`

Reply:
Can I check, what do you mean by being shadowed?

Reply:
https://stackoverflow.com/questions/26297180/what-are-shadow-variables-in-java

Reply:
👍

Reply:
Sorry I missed the Guava part in the previous review, will add that and backend validation in accordingly.